examples/vault/get.go

```diff
@@ -0,0 +1,62 @@
+// vault rotate is an example of how to use the rotate method
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/pangeacyber/pangea-go/pangea-sdk/v2/pangea"
+	"github.com/pangeacyber/pangea-go/pangea-sdk/v2/service/vault"
+	"github.com/pangeacyber/pangea-go/pangea-sdk/v2/service/audit"
+)
+
+func main() {
+	token := os.Getenv("PANGEA_VAULT_TOKEN")
+	if token == "" {
+		log.Fatal("Unauthorized: No token present")
+	}
+
+	auditTokenID := os.Getenv("PANGEA_AUDIT_TOKEN_ID")
+	if auditTokenID == "" {
+		log.Fatal("Unauthorized: No audit token id present")
+	}
+    vaultConfig := pangea.Config{
+		Token:  token,
+		Domain: os.Getenv("PANGEA_DOMAIN"),
+	}
+	vaultObj := vault.New(&vaultConfig)
+
+	ctx := context.Background()
+
+	fmt.Println("Fetch the audit token...")
+	getRequest := &vault.GetRequest{
+        ID: auditTokenID,
+	}
+	storeResponse, err := vaultObj.Get(ctx, getRequest)
+	if err != nil {
+		log.Fatal(err)
+	}
+    auditToken := storeResponse.Result.CurrentVersion.Secret
```

Comment:
<img width="718" alt="image" src="https://github.com/pangeacyber/pangea-go/assets/111303288/2cc069d5-7e7c-46c3-a72d-f710980db048">
Is indentation wrong or it's just github when displaying it?